frontend/awx/administration/management-jobs/hooks/useManagementJobColumns.tsx

```diff
@@ -1,33 +1,40 @@
 import { useMemo } from 'react';
-import { ITableColumn, usePageNavigate } from '../../../../../framework';
-import {
-  useCreatedColumn,
-  useDescriptionColumn,
-  useModifiedColumn,
-  useNameColumn,
-} from '../../../../common/columns';
+import { ITableColumn, useGetPageUrl } from '../../../../../framework';
 import { SystemJobTemplate } from '../../../interfaces/SystemJobTemplate';
 import { AwxRoute } from '../../../main/AwxRoutes';
+import { TextCell } from '../../../../../framework';
+import { useTranslation } from 'react-i18next';
 
-export function useManagementJobColumns(options?: {
-  disableSort?: boolean;
```

Comment:
since I removed the parameters here, didn't use them here @prat98 
`const toolbarFilters = useManagementJobFilters();`

let me know if this isn't the correct way

Reply:
Ok that makes sense here